helm/hpcc/templates/thor.yaml

```diff
@@ -53,6 +54,7 @@ data:
 {{- include "hpcc.generateLoggingConfig" (dict "root" .root "me" $thorScope) | indent 6 }}
 {{- include "hpcc.generateTracingConfig" (dict "root" .root "me" $thorScope) | indent 6 }}
 {{ include "hpcc.generateVaultConfig" . | indent 6 }}
+{{ include "hpcc.generateAnalyzerOptions" (dict "root" .root "me" $thorScope) | indent 6 }}
```

Comment:
Shouldn't you also be generating this elsewhere, e.g. eclagent?

Reply:
should also update values.schema.json to show that it can be defined at the component level.

Reply:
I think Jake's first comment still applies.

Reply:
The changes to thor.yaml weren't showing up.  After rebasing, the changes on lines 55, 65 and 70 showing now.

Reply:
>Shouldn't you also be generating this elsewhere, e.g. eclagent?

Isn't this still true? i.e. not just for thor.yaml, but also for eclagent.yaml

Reply:
Line 70 generates AnalyzerOptions for eclagent.  I've confirmed that EclAgent sees the AnalyzerOptions in its component configuration query.

Reply:
what about non-thor eclagent (hthor - roxie-workunit) ?

Reply:
Analyzer doesn't execute for hThor job at the moment.  I've create a jira to look into having the analyzer run for hThor jobs: https://hpccsystems.atlassian.net/browse/HPCC-34143